centml/sdk/api.py

```diff
@@ -48,6 +48,7 @@ def get_compute(id):
 def create_inference(
     name, image, port, is_private, hw_to_id_map, health, min_replicas, max_replicas, env, command, command_args, timeout
 ):
+    triplet = None
```

Comment:
**Note** fixed triplet variable issue as its not accessible in inference deployment request if is_private is false and its throwing error if the is_private is false, after defining it outside the condition it will accessible in the below object  where request payload is getting created